ietf/templates/nomcom/reclassify_feedback_item.html

```diff
@@ -0,0 +1,103 @@
+                               aria-label="Unclassified"
+                               title="Unclassified">
+                    </td>
+                    {% for t in types %}
+                        <td class="text-center">
+                            <input type="radio"
+                                   class="form-check-input"
+                                   name="type"
+                                   value="{{ t.slug }}"
+                                   id="{{ t.name|slugify }}"
+                                   aria-label="{{ t.name }}"
+                                   {% if feedback.type == t %}checked{% endif %}
+                                   title="{{ t.name }}">
+                        </td>
+                    {% endfor %}
+                    <td>{{ feedback.author }}</td>
+                    <td>{{ feedback.subject }}</td>
+                    <td>
+                        <button type="button"
+                                class="btn btn-primary btn-sm"
+                                data-bs-toggle="modal"
+                                data-bs-target="#modal{{ feedback.id }}">
+                            View
+                        </button>
+                        <div class="modal fade"
+                             id="modal{{ feedback.id }}"
+                             tabindex="-1"
+                             role="dialog"
+                             aria-labelledby="label{{ feedback.id }}"
+                             aria-hidden="true">
+                            <div class="modal-dialog modal-dialog-scrollable modal-xl">
+                                <div class="modal-content">
+                                    <div class="modal-header">
+                                        <p class="h5 modal-title" id="label{{ feedback.id }}">{{ feedback.subject }}</p>
+                                        <button type="button"
+                                                class="btn-close"
+                                                data-bs-dismiss="modal"
+                                                aria-label="Close"></button>
+                                    </div>
+                                    <div class="modal-body">
+                                        <pre>{% decrypt feedback.comments request year 1 %}</pre>
+                                    </div>
+                                    <div class="modal-footer">
+                                        <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+        <input type="hidden" name="feedback_id" value="{{ feedback.id }}">
+        <button class="btn btn-primary" type="submit">Classify</button>
```

Comment:
I think the button should perhaps say "Save" rather than "Classify"?

Reply:
IDK, the form for pending email says "Classify", and this form is reached by the "Reclassify" button.

Reply:
Ok - I guess my issue is with the existing UI then. :-) Leave it matching.